cli_tools/gce_windows_upgrade/upgrader/upgrader.go

```diff
@@ -139,9 +148,60 @@ func (u *upgrader) init() error {
 	if err != nil {
 		return daisy.Errf("Failed to create GCE client: %v", err)
 	}
+
+	err = u.initLogger()
+	if err != nil {
+		return daisy.Errf("Failed to initialize logger: %v", err)
+	}
+
 	return nil
 }
 
+func (u *upgrader) initLogger() error {
```

Comment:
After I finished reading this function, I wondered: "Where is daisy logging being initialized"

Reply:
removed all log changes